Documentation/install/vmware/vmware-terraform.md

```diff
@@ -125,26 +126,24 @@ Next, deploy the cluster:
 $ terraform apply ../../platforms/vmware
 ```
 
-Wait for `terraform apply` until all tasks complete. Tectonic cluster should be ready upon completion of apply command. If any issues arrise please check the known issues and workarounds below.
+Wait for `terraform apply` to complete all tasks. The Tectonic cluster should be ready upon completion of the `apply` command. If any issues arise see the [troubleshooting][troubleshooting] guide.
```

Comment:
second sentence: replace with: "See the [troubleshooting][troubleshooting] guide if problems occur."